src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/MsaPassthroughHandler.cs

```diff
@@ -31,20 +31,15 @@ internal class MsaPassthroughHandler : IMsaPassthroughHandler
             _msaPlugin = msaPlugin;
             _wamProxy = wamProxy;
             _parentHandle = parentHandle;
-        }
-
-        public bool IsPassthroughEnabled(AuthenticationRequestParameters authenticationRequestParameters)
-        {
-            // TODO: change this to a config object
-            bool enabled = authenticationRequestParameters.ExtraQueryParameters.TryGetValue("msal_msa_pt", out string val) &&
-                string.Equals("1", val);
-            _logger.Info("WAM MSA-PT: PassThrough is enabled? " + enabled);
-
-            return enabled;
-        }
+        }      
 
         public async Task<string> FetchTransferTokenAsync(AuthenticationRequestParameters authenticationRequestParameters, WebAccountProvider accountProvider)
         {
+            if (!authenticationRequestParameters.AppConfig.IsMsaPassthrough)
+            {
```

Comment:
so you're moving to the API change? I kind of like the extra qp because it's such a small use case.

Reply:
Yeah, but it is hidden behind an obscure namespace.